### DISCUSSION TITLE
Validations on Rotor 37 compressor

### DISCUSSION THREAD

**Discussion by 1064168551** [mdolab/dafoam#460] [Tutorials]
1. What is the reference for this example if I want to cite this adjoint optimization case in my paper. 
2. And where can I find the Validations on Rotor 37 compressor via dafoam?

**Comment by raisePiginYourhome** [comment]
have you run this instance?

**Comment by friedenhe** [comment]
> 1. What is the reference for this example if I want to cite this adjoint optimization case in my paper.
> 2. And where can I find the Validations on Rotor 37 compressor via dafoam?

1. No need to cite this specific case. Please just cite the DAFoam papers: https://dafoam.github.io/mydoc_get_started_faq.html#how-to-cite-dafoam
2. We haven't validated the Rotor 37 case using DAFoam yet. Please feel free to share your validation results.

BTW: we recently enabled using the totalPressure and totalTemperature as the BCs for the Rotor 37 case. Please check the latest runScript.py
